### PR TITLE
version 0.1 various args

### PR DESCRIPTION
**Pull request by bmk101**

### CODE REVIEW

**Comment by bmk101** [comment]
A simple text_Morse converter. Python 3.x

File: chall02/rgallard.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import sys
```
Comment (anna-dominguez) [inline chall02/rgallard.py:1]: A shebang is missing

File: chall02/rgallard.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+morse = {
```
Comment (anna-dominguez) [inline chall02/rgallard.py:3]: Better to write globals in upper case

File: chall02/rgallard.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	'U':'..-',
+	'V':'...-',
+	'W':'.--',
+	'X':'-..-',
+	'Y':'-.--',
+	'Z':'--..',
+	' ':' '
+	}
+
+i = 1
```
Comment (anna-dominguez) [inline chall02/rgallard.py:33]: Why ?

File: chall02/rgallard.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+j = 0
```
Comment (anna-dominguez) [inline chall02/rgallard.py:34]: Why ?

File: chall02/rgallard.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+talk = ""
+
+if (2 > len(sys.argv)):
+	talk = "<a-zA-Z string>"
+while (i < len(sys.argv)):
```
Comment (anna-dominguez) [inline chall02/rgallard.py:39]: Why use index in Python 😱 it's not C, you don't need to iterate through your array or string with an index, you can use `for` as you did it below

File: chall02/rgallard.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+	for j in sys.argv[i]:
```
Comment (anna-dominguez) [inline chall02/rgallard.py:40]: You didn't need to declare `j` before

File: chall02/rgallard.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+		j = j.upper()
+		if (j in morse.keys()):
```
Comment (anna-dominguez) [inline chall02/rgallard.py:42]: Good 👍 it's a better way to check if it's a writable character than use the functions `isascii` or `isalpha` but it makes you declare a variable in order to store your result when you could write if you've done the checks before

**Review by anna-dominguez — COMMENTED** [review]
Well done not forgetting to send an executable. But where is you comment explaining your code ??

**Pull request closed without merging**